### ISSUE TITLE
New terms for measurements branch

### ISSUE THREAD

**Issue by s-carney** [EBISPOT/efo#1523]
Hi EFO, 

I have attached suggestions of terms (and xrefs in some cases) for the measurements branch.

CC: @paolaroncaglia 

[Terms for measurements branch.xlsx](https://github.com/EBISPOT/efo/files/8538603/Terms.for.measurements.branch.xlsx)

**Comment by paolaroncaglia** [comment]
@s-carney @dosumis cc @zoependlington 
Thank you Sharon for following up on your EFO-users email by opening this ticket.
As your new term requests are for measurement terms, I'll defer this to @dosumis please, so he may comment on feasibility and timeline, and on whether this ticket might be in scope for the future review of the EFO measurement branch. Thanks.

**Comment by s-carney** [comment]
Thanks Paola!

**Comment by jamesamcl** [comment]
### Automated triage — 2026-08-26

**Status: stale premise, partially delivered. Recommend re-scoping rather than actioning as filed.**

**Ticket state.** Opened 2022-04-22 by @s-carney, deferred to @dosumis pending "the future review of the EFO measurement branch", no maintainer response since. The attached spreadsheet **still resolves** and contains **236 requested terms** with suggested parents, definitions and NCIT/SNOMEDCT xrefs.

**What has happened since.** The review this ticket was waiting on is done: epic #787 closed 2026-08-04, and the v3.77.0 OBA migration obsoleted **712 EFO terms with OBA replacements (597 measurement-labelled)**. New measurement terms are now normally requested in **OBA**, not created in EFO (cf. #2683, #2740). So the original ask — bulk-create 236 measurement terms in EFO — is no longer the right shape.

**Coverage check against `efo-edit.owl` today:**

- **97 of 236 (41%) already exist and are live** — added via other routes over the past four years (e.g. `amylase measurement` EFO_0803343, `cholesterol in large LDL measurement` EFO_0021901).
- **None of the 97 were obsoleted** by the OBA migration, but **57 of them carry the comment "To be reviewed for inclusion in OBA"** — so that portion is itself provisional.
- **139 remain absent.**

**The outstanding 139, by block:**

| Block | Count | Suggested destination |
|---|---:|---|
| UK Biobank serology "… antigen level measurement" | 44 | **OBA** — no equivalents found in OLS; needs an upstream request |
| Misc clinical chemistry | 35 | mixed — some OBA equivalents exist (globulin, urine creatinine), others not (direct bilirubin, reticulocyte percentage) |
| Lipoprotein / NMR metabolomics | 28 | **mostly synonyms on existing EFO terms** (see below) |
| Body composition & impedance | 15 | **OBA** — no equivalents found |
| Clinical risk / questionnaire scores | 12 | **not OBA** (not biological attributes); SNOMED has several |
| Pulmonary function (post-bronchodilation) | 5 | to review |

**Cheapest win first:** at least **17 of the 28 lipoprotein rows are not new terms at all** — they differ from an existing EFO term only in label wording:

- requested `concentration of large LDL particles` → existing **`concentration of large LDL particles measurement`** (EFO_0022160)
- requested `total lipids in medium LDL measurement` → existing **`total lipids in medium LDL`** (EFO_0022180)
- requested `concentration of chylomicrons and extremely large VLDL particles` → existing (EFO_0022260)

These could be closed out as `hasExactSynonym` additions rather than new classes.

**Suggested disposition:**

1. Keep this issue open as the record of the residual 139, but relabel and drop the implied bulk-create framing.
2. Split the 44 serology and 15 body-composition terms into an **upstream OBA request**.
3. Handle the ~17 lipoprotein label variants as synonym additions in EFO.
4. Decide separately on the 12 clinical risk scores — not biological attributes, so OBA is not the right home; they may be out of scope for EFO too.

Closing in favour of per-block refiling would also be reasonable, but doing so without recording the residual 139 somewhere would lose the only trace of them.

---
*This is an automated triage review by an AI agent; maintainers, please correct if anything here is wrong.*
